backend/migrations/2023-07-01-152628_squashed_setup/up.sql

```diff
@@ -176,93 +176,93 @@ END;
 $$;
 
 CREATE TABLE layers (
-    id integer NOT NULL,
-    map_id integer NOT NULL,
-    type layer_type NOT NULL,
-    name text NOT NULL,
-    is_alternative boolean NOT NULL
+    id INTEGER NOT NULL,
+    map_id INTEGER NOT NULL,
+    type LAYER_TYPE NOT NULL,
+    name TEXT NOT NULL,
+    is_alternative BOOLEAN NOT NULL
 );
 
 CREATE SEQUENCE layers_id_seq
-AS integer
+AS INTEGER
 START WITH 1
 INCREMENT BY 1
 NO MINVALUE
 NO MAXVALUE
 CACHE 1;
 
 CREATE TABLE maps (
-    id integer NOT NULL,
-    name text NOT NULL,
-    creation_date date NOT NULL,
-    deletion_date date,
-    last_visit date,
-    is_inactive boolean NOT NULL,
-    zoom_factor smallint NOT NULL,
-    honors smallint NOT NULL,
-    visits smallint NOT NULL,
-    harvested smallint NOT NULL,
-    privacy privacy_option NOT NULL,
-    description text,
+    id INTEGER NOT NULL,
+    name TEXT NOT NULL,
+    creation_date DATE NOT NULL,
+    deletion_date DATE,
+    last_visit DATE,
+    is_inactive BOOLEAN NOT NULL,
+    zoom_factor SMALLINT NOT NULL,
+    honors SMALLINT NOT NULL,
+    visits SMALLINT NOT NULL,
+    harvested SMALLINT NOT NULL,
+    privacy PRIVACY_OPTION NOT NULL,
```

Comment:
This is not according to current database guidelines, we wrote: "use `snake_case` for everything except database keywords, which should be `UPPERCASE`". So it should be `privacy_option`. Isn't this supported by sqlfluff?

Btw. even for INTEGER it is questionable if it is a keyword, according to https://www.postgresql.org/docs/current/sql-keywords-appendix.html INTEGER is non-reserved (cannot be function or type).

Reply:
> This is not according to current database guidelines, we wrote: "use `snake_case` for everything except database keywords, which should be `UPPERCASE`". So it should be `privacy_option`. Isn't this supported by sqlfluff?

No, sadly it's not, but I applied it for now by manually changing the rules.
https://docs.sqlfluff.com/en/stable/rules.html#rule-capitalisation.types


> Btw. even for INTEGER it is questionable if it is a keyword, according to https://www.postgresql.org/docs/current/sql-keywords-appendix.html INTEGER is non-reserved (cannot be function or type).

I could make it snake_case, but since it's a standard keyword, I would keep it uppercase.